lib/screens/order_history.dart

```diff
@@ -95,3 +114,12 @@ class _OrderList extends State<OrderList> {
     Navigator.push(context, orderResponse);
   }
 }
+
```

Comment:
Instead of creating a new class for Order model, can you add these fields in the existing Order model in the location models/order.dart. Please don't make them required as it mayb affect other files already using this class.

Reply:
Sure, Let me do that